commit 0f8e617b8f215561ec8a9d735d02bac8b27f602d
Author: Benoit Pierre <benoit.pierre@gmail.com>
Date:   Wed May 11 15:04:16 2016 +0200

    gui/log_dbus: fix possible crash
    
    pynotify uses gtk, and importing gtk before wx can trigger a crash:
    see http://trac.wxwidgets.org/ticket/15898

diff --git a/plover/gui/log_dbus.py b/plover/gui/log_dbus.py
index c0a2e0f..fcc724a 100644
--- a/plover/gui/log_dbus.py
+++ b/plover/gui/log_dbus.py
@@ -1,8 +1,15 @@
 
-from plover import log, __name__ as __software_name__
-import pynotify
 import logging
 
+# Fix a possible crash...
+# See: http://trac.wxwidgets.org/ticket/15898
+import gtk
+gtk.remove_log_handlers()
+
+import pynotify
+
+from plover import log, __name__ as __software_name__
+
 
 pynotify.init(__software_name__.capitalize())
 
